core/domain/suggestion_services_test.py

```diff
@@ -137,6 +138,51 @@ def test_create_new_suggestion_successfully(self):
             self.assertDictContainsSubset(
                 expected_suggestion_dict, observed_suggestion.to_dict())
 
+    def test_cannot_create_suggestion_with_invalid_suggestion_type(self):
+        with self.assertRaisesRegexp(Exception, 'Invalid suggestion type'):
+            suggestion_services.create_suggestion(
+                'invalid_suggestion_type',
+                suggestion_models.TARGET_TYPE_EXPLORATION,
+                self.target_id, self.target_version_at_submission,
+                self.author_id, self.change, 'test description',
+                self.reviewer_id)
+
+    def test_get_all_stale_suggestions(self):
+        suggestion_services.create_suggestion(
+            suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            self.author_id, self.change, 'test description',
+            self.reviewer_id)
+
+        with self.swap(
+            suggestion_models, 'THRESHOLD_TIME_BEFORE_ACCEPT_IN_MSECS', 0):
+            self.assertEqual(
+                len(suggestion_services.get_all_stale_suggestions()), 1)
+
+        with self.swap(
+            suggestion_models, 'THRESHOLD_TIME_BEFORE_ACCEPT_IN_MSECS',
+            7 * 24 * 60 * 60 * 1000):
```

Comment:
This is looking just like the value it is set to right? Why do we need to swap it?

Reply:
I just added it show explicitly that there are no stale suggestions if I put THRESHOLD_TIME_BEFORE_ACCEPT_IN_MSECS to its normal value.
> Can you just confirm that all the success cases already have corresponding tests? (As most of the cases here test the failures right?)

The coverage is at 100% so I guess every success cases are already covered.

Reply:
Hmmm, I agree it is better for readability. Let's go with this itself.